src/main/java/net/openhft/chronicle/threads/ExceptionHandlerStrategy.java

```diff
@@ -28,7 +29,7 @@ static ExceptionHandlerStrategy strategy() {
     class LogAndRemove implements ExceptionHandlerStrategy {
         @Override
         public boolean handle(EventLoop eventLoop, EventHandler handler, Throwable t) {
-            if (!(t instanceof InvalidEventHandlerException)) {
+            if (!(t instanceof InvalidEventHandlerException || t instanceof ClosedIllegalStateException)) {
```

Comment:
cool. Need to reflect this change in doco for e.g. `net.openhft.chronicle.core.threads.EventHandler#loopFinished`

Reply:
I am afraid this would eventually lead to more special cases. Is there another way to solve the problem?